### DISCUSSION TITLE
which servos does servo autotrim trim?

### DISCUSSION THREAD

**Discussion by TTGG4400** [iNavFlight/inav#7153] [Q&A] [answered]
There's an INAV doc, shown below, that says servo autotrim trims servos 2-5. It may be out of date, or using a diff numbering system - anyone know the answer for INAV3.0.0?  
I use OTX CHS 1-4 for AERT - is why I'm asking.  And on the INC outputs tab roll and pitch are servos 4 and 3.

I'd like to know that if I turn on servo autotrim - that the correct servos will get trimmed.  Especially since I got mode on CH5.

![image](https://user-images.githubusercontent.com/85150615/122676225-27aa0a00-d1ab-11eb-898f-3d37186b660a.png)

![image](https://user-images.githubusercontent.com/85150615/122677116-e61b5e00-d1ae-11eb-81fc-bd9f9ff7ec47.png)

**Comment by avsaase** [comment, marked as answer]
This was changed in 3.0. Only servos with a "stabilized" rule are trimmed. In your setup this means servos 3, 4 and 6.

**Reply by avsaase** [reply]
The docs need to be updated to reflect these changes.

**Reply by TTGG4400** [reply]
Thx much.

**Reply by TTGG4400** [reply]
One last note - I updated the doc ... 
![image](https://user-images.githubusercontent.com/85150615/122685263-538fb480-d1d8-11eb-9516-33381546d7f2.png)
